recipes/turtle/all/conanfile.py

```diff
@@ -24,6 +24,8 @@ def requirements(self):
 
     def package_id(self):
         self.info.clear()
+        self.cpp_info.libdirs = []
```

Comment:
Hi @Cogitri - thanks for your contribution.

Indeed these two lines are advised for header only library - however they should be placed in the `package_info()` method, rather than the `package_id()` one.

Reply:
Ah thanks :)